pykusto/expressions.py

```diff
@@ -469,25 +478,26 @@ def __len__(self) -> NumberExpression:
     def array_length(self) -> NumberExpression:
         return NumberExpression(KQL('array_length({})'.format(self.kql)))
 
-    def contains(self, other: ExpressionType) -> 'BooleanExpression':
+    def array_contains(self, other: ExpressionType) -> 'BooleanExpression':
         return BooleanExpression.binary_op(other, ' in ', self)
 
+    def contains(self, other: ExpressionType) -> 'BooleanExpression':
```

Comment:
Why using "contains" for both array_contains and string_contains if we can use "in" for array_contains and "contains" for string_contains and avoid all the mess?

Reply:
mess for example: ValueError inside AnyTypeColumn

Reply:
Once a `contains` method exists in `StringExpression`, it will also exist in `AnyTypeColumn` and will be prone to misuse by someone thinking it can be used for arrays.
A valid solution is to eliminate the `contains` method altogether, and have only the methods `string_contains` and `array_contains`.
However, I was trying to save the user some typing in case the column type is known, so they can just use `contains` and have the type system choose the correct method. The only downside I can see is having the method that raises a `ValueError` in `AnyTypeColumn`. What other issues do you see?
Do you think it's better to eliminate the `contains` method altogether?

Reply:
After discussion: we will go with "contains" for strings and "in" / "array_contains" for arrays